Documentation/boot-flow.md

```diff
@@ -48,13 +48,19 @@ In the second stage, we are all set, and depending on the platform, we could
 already run the final OS, a hypervisor, or any other payload. However, some
 platforms require extra steps. For this example, we will look at RISC-V.
 
-RISC-V describes [SBI](https://github.com/riscv-non-isa/riscv-sbi-doc), the
-Supervisor Binary Interface, in the [privileged spec](https://riscv.org/technical/specifications/). The role of SBI is to set up or
-delegate exception and interrupt handling, register functions for the OS to call
-into, and drop into a lower privileged mode named S-mode which makes an [MMU](https://en.wikipedia.org/wiki/Memory_management_unit) available, and finally
-execute the operating system. This is where we hand over to LinuxBoot, our
-boot loader environment. In oreboot, we implement SBI via [RustSBI](https://github.com/rustsbi/rustsbi), a Rust crate offering all the necessary
-functions we need as a library.
+RISC-V mentions SBI, the Supervisor Binary Interface, in the ISA spec Volume 2,
+[Privileged Specification](https://riscv.org/technical/specifications/). It now
+evolves as [SBI specifcation](https://github.com/riscv-non-isa/riscv-sbi-doc).
+
+The role of SBI is to set up or delegate exception and interrupt handling,
+register functions for the OS to call into, and drop into a lower privileged
+mode named S-mode. While having less access to the hardware, it makes an [MMU](https://en.wikipedia.org/wiki/Memory_management_unit) with [paged virtual memory](https://www.sifive.com/blog/all-aboard-part-9-paging-and-mmu-in-risc-v-linux-kernel)
+available. RISC-V complements this in the (UNIX-class) [platform specification](https://github.com/riscv/riscv-platform-spec).
```

Comment:
this link is dead

Reply:
Whoopsies, good catch - thank you! :fishing_pole_and_fish:

Reply:
... that'll get a fixup then.

Do you have access enough to request changes on a PR? :grimacing: